### ISSUE TITLE
Should this be published on the NodeSchool site?

### ISSUE THREAD

**Issue by martinheidegger** [toolness/security-adventure#6]
@toolness I am just wondering if this adventure should be added to the NodeSchool site?

**Comment by sericaia** [comment]
I totally agree. I was planning to add the same issue! Can you add it here nodeschool.io ? Much more people will get into it! Thanks

**Comment by toolness** [comment]
Hey sorry I didn't reply to this! I'd like to add it to the nodeschoo.io site... the one thing I'm not sure about is that the security adventure is a little bit unfinished, because I originally wanted to have the tutorial teach more stuff that I don't actually know much about (e.g. how to securely store passwords). I guess I could just publish this as-is and potentially make a "Chapter 2" later on?
